src/Settings/SettingsHelper.php

```diff
@@ -354,4 +548,42 @@ public function getIntegrationFieldsValue(array $dbSettingsValue, array $formFie
 
 		return $formFields;
 	}
+
+	/**
+	 * Create our custom tables, used on plugin activation.
+	 *
+	 * @return void
+	 */
+	public function createDbTables(): void
+	{
+		global $wpdb;
+
+		$tableNameSettings = "{$wpdb->prefix}" . Config::getDbSettingsName();
+		$tableNameOptions = "{$wpdb->prefix}" . Config::getDbOptionsName();
+
+		$sqlSettings = "CREATE TABLE {$tableNameSettings} ( 
+			meta_id BIGINT(20) UNSIGNED NOT NULL AUTO_INCREMENT,
+			post_id BIGINT(20) UNSIGNED NOT NULL DEFAULT '0',
+			meta_key VARCHAR(255) NULL DEFAULT NULL,
+			meta_value LONGTEXT NULL DEFAULT NULL,
```

Comment:
What kind of data is going to be stored here? Serialized arrays, JSON, something else?

Reply:
basicly any kind. It can be string or Serialized arrays, JSON etc

Reply:
Yeah, I'm not too happy about that. Custom DB tables should be custom following all the best practices (normal forms etc.) of database design.

Just copy-pasting meta and option tables make little sense to me tbh, plus you're losing all the caching WP offers.